caim_base/static/css/normalize.css

```diff
@@ -0,0 +1,351 @@
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
```

Comment:
is this file only intended for use in the pdfs or should we be pulling this into the stylesheets for all of our pages?

Reply:
That's a good question, I believe Bootstrap includes css normalization already so we shouldn't need it anywhere we're using that.